fix the return value of route53_zone

prasadkatti opened the pull request:
##### SUMMARY
All the values currently documented as return values are returned inside a 'result' key. So if you registered the output of the task as 'output', then you would need to do 'output.result.zone_id' instead of 'output.zone_id'.

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
`route53_zone`

##### ANSIBLE VERSION
<!--- Paste verbatim output from "ansible --version" between quotes below -->
```
devel
```

willthames commented:
Oh, I hoped from your comment that you were going to fix the return value, rather than the docs.

Might be worth leaving docs as is and just passing `**result` at the end of the `exit_json`. We'll need to keep the existing `result` key but at least it will be more obvious to users.

Not sure if there's a way to deprecate return results, unfortunately!

prasadkatti commented:
oh sorry, I see it too now. That is a really bad commit message. :(

I will make suggested changes.

prasadkatti committed:
fix the return value of route53_zone

All the values currently documented as return values are returned inside a 'result' key.
So if you registered the output of the task as 'output', then you would need to do 'output.result.zone_id' instead of 'output.zone_id'.
This commit fixes that so that you can do 'output.zone_id'.

prasadkatti commented:
@willthames - ready_for_review
